scripts/deployContracts.ts

```diff
@@ -226,25 +227,38 @@ export async function deployRootContracts(
         });
         getLogger('Deployer').info('🤞 InflationController');
 
+        let tx = await sqtToken.setMinter(inflationController.address);
+        await tx.wait(confirms);
+
         //deploy vesting contract
         const vesting = await deployContract<Vesting>('Vesting', 'root', { deployConfig: [deployment.root.SQToken.address] });
         getLogger('Deployer').info('🤞 Vesting');
 
         //deploy PolygonDestination contract
         const polygonDestination = await deployContract<PolygonDestination>('PolygonDestination' as any, 'root',
-            { deployConfig: [settingsAddress, constants.AddressZero] });
+            { deployConfig: [settingsAddress, _wallet.address] });
+
+        let rootChainManager;
+        if (network === 'local') {
+            // deploy MockRootChainManager
+            rootChainManager = await new RootChainManager__factory(wallet).deploy();
+        }
+        // tx = await inflationController.setInflationDestination(polygonDestination.address);
```

Comment:
Need to uncomment this?

Reply:
No, it will break some inflation tests which expects a normal wallet.

Reply:
For the mainnet, we can set that after the contract creation.